scipy/stats/_continuous_distns.py

```diff
@@ -8395,7 +8395,7 @@ def f(x):
     def _stats(self, chi):
         chi2 = chi**2
         phi = _argus_phi(chi)
-        m = np.sqrt(np.pi/8) * chi * np.exp(-chi2/4) * sc.iv(1, chi2/4) / phi
+        m = np.sqrt(np.pi/8) * chi * sc.ive(1, chi2/4) / phi
```

Comment:
Defined as:
```
ive(v, z) = iv(v, z) * exp(-abs(z.real))
```
So:
```
ive(1, chi2/4) = iv(1, chi2/4) * exp(-abs(chi2/4.real))
```
Assuming `chi` is real to begin with `chi2` is real and positive, so looks good.